src/System.Net.NetworkInformation/src/System/Net/NetworkInformation/IPGlobalProperties.cs

```diff
@@ -61,7 +61,6 @@ internal static IPGlobalProperties InternalGetIPGlobalProperties()
 
         public abstract IPGlobalStatistics GetIPv6GlobalStatistics();
 
-        /// Returns a list of all unicast IP addresses after ensuring they are all stable.
```

Comment:
Sure. If you want Intellisense, you need to use the XML tags, otherwise VS just thinks it's a regular comment, as far as I have ever seen. TBH, it's unclear to me whether our long-term plan is to use the in-code XML docs as our intellisense files, but I know we are shipping them as such currently.